packages/grafana-eslint-rules/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "@grafana/eslint-plugin",
+  "description": "ESLint rules for use within the Grafana repo. Not suitable (or supported) for external use.",
+  "version": "0.0.1",
```

Comment:
not sure how lerna with react to this one. better if you version it to the exact same version as all of the other packages.

Reply:
Also, missing license and author